upup/pkg/fi/vfs_castore.go

```diff
@@ -754,6 +754,21 @@ func (c *VFSCAStore) DeleteSecret(item *KeystoreItem) error {
 		p := c.buildSSHPublicKeyPath(item.Name, item.Id)
 		return p.Remove()
 
+	case SecretTypeKeypair:
+		version, ok := big.NewInt(0).SetString(item.Id, 10)
+		if !ok {
+			return fmt.Errorf("keypair had non-integer version: %q", item.Id)
+		}
+		p := c.buildCertificatePath(item.Name, version)
+		if err := p.Remove(); err != nil {
+			return err
```

Comment:
Do we need fmt?

Reply:
We use fmt when we are adding information to the error.  Kinda marginal here, but I'll add it...

We need `fmt.Errorf` when we're returning a third party error we can't control.  But if we wrap all our errors we end up with:

`cannot do X: failed to do X: unable to X: error while doing X: <real error>`